### PR TITLE
DashboardScene: Inspect / Json tab

### PR DESCRIPTION
**Pull request by torkelo**
* [x] Found that sharing underlying react code was too messy as there were too many differences (but we are sharing some util functions) 
* [x] Showing panel json
* [x] Showing panel data
* [x] Showing raw data frames 
* [x] filter out scopedVars _sceneObject in JSON
* [x] Unit tests
* [x] Add same tracking as current impl

### CODE REVIEW

File: public/app/features/inspector/utils/utils.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { AppEvents } from '@grafana/data';
```
Comment (dprokop) [inline public/app/features/inspector/utils/utils.ts:1]: Maybe we can move those utils to their destination - dashboard-scene/inspect and reference in the old code if needed?

File: public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx [hunk 2]
```diff
@@ -1,11 +1,264 @@
...
+import { getPrettyJSON } from 'app/features/inspector/utils/utils';
+import { reportPanelInspectInteraction } from 'app/features/search/page/reporting';
 
-import { InspectTabState } from './types';
+import { PanelRepeaterGridItem } from '../scene/PanelRepeaterGridItem';
+import { buildGridItemForPanel } from '../serialization/transformSaveModelToScene';
+import { gridItemToPanel } from '../serialization/transformSceneToSaveModel';
+import { getDashboardSceneFor, getPanelIdForVizPanel } from '../utils/utils';
+
+export type ShowContent = 'panel-json' | 'panel-data' | 'data-frames';
```
[the remark this answers is not in the document]
Comment (dprokop) [reply public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx:35]: Maybe move it to the new inspect, and reference back in the old code.

**Review by dprokop — APPROVED** [review]
Looks good, I would just move what's reused to the new implementation and reference it back in the current inspect. It's gonna be easier to get rid of it at some point thanks to that.

Other than that - cosmetic comments to consider.

File: public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx:35]: @dprokop 
After discussing in #frontend this is actually better than an enum for many scenarios where you want string values

File: public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx [hunk 3]
```diff
@@ -1,11 +1,264 @@
...
+          'Raw data without transformations and field config applied. '
+        ),
+        value: 'data-frames',
+      });
+    }
+
+    return options;
+  }
+
+  public onChangeShow = (value: SelectableValue<ShowContent>) => {
```
[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx:98]: @dprokop ah, good point, the show naming was taken from the current code :)

File: public/app/features/inspector/utils/utils.ts (hunk 1 above)
Comment (torkelo) [reply public/app/features/inspector/utils/utils.ts:1]: @dprokop maybe, I think we can do this later the features/inspector is a bit messy now with shared code with old dashboard & explore, so leaving it here for now I think.

File: public/app/features/dashboard-scene/inspect/InspectJsonTab.test.tsx [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+          y: 0,
+          width: 10,
+          height: 12,
+          body: panel,
+        }),
+      ],
+    }),
+  });
+
+  activateFullSceneTree(scene);
```
[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/features/dashboard-scene/inspect/InspectJsonTab.test.tsx:152]: good idea, will try to move it to scenes in the future

File: public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (torkelo) [reply public/app/features/dashboard-scene/inspect/InspectJsonTab.tsx:34]: but I am torn myself on what is best (and makes the code the clearest)

**Pull request merged**